ztp/resource-generator/entrypoints/generator

```diff
@@ -0,0 +1,425 @@
+
+  podman run --rm -v \$host_dir:\$container_dir:Z,U \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} \$container_dir/policygentemplates
+  podman run --rm -v \$host_dir:\$container_dir:Z,U \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} policygentemplates out/generated_configCRs
+  podman run --rm -v \$host_dir:\$container_dir:Z,U \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} policygentemplates/mycluster.yaml out/generated_configCRs
+
+  # when DEST_DIR is not given, or relative SRC_PATH and DEST_DIR are given but \$container_dir is other than /home/ztp_resources,
+  # the container environment variable MOUNT_DIR needs to be given with podman flag "-e"
+  podman run --rm -v \$host_dir:\$container_dir:Z,U -e MOUNT_DIR=\$container_dir \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} \$container_dir/policygentemplates
+  podman run --rm -v \$host_dir:\$container_dir:Z,U -e MOUNT_DIR=\$container_dir \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} policygentemplates/mycluster.yaml out/generated_configCRs
+  podman run --rm -v \$host_dir:\$container_dir:Z,U -e MOUNT_DIR=\$container_dir \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} policygentemplates out/generated_configCRs
+
+  # generate non-policy wrapped CRs
+  podman run --rm -v \$host_dir:\$container_dir:Z,U \$THIS_CONTAINER ${PROG} ${SUB_COMMAND} -w false \$container_dir/policygentemplates \$container_dir/out/generated_configCRs
+
+  Note: With podman, to grant the user running in the container permission to read and write the folder mounted in from the host,
+        add :U volume option to the "-v" flag or run the container as root with the flag "-u root".
+
+        The above examples are oneshot use. If you want to run commands interactively, you will need to pass the "-it" flag to podman:
+        podman run --rm -it -v \$host_dir:\$container_dir:Z,U -e MOUNT_DIR=\$container_dir \$THIS_CONTAINER bash
+EOF
+}
+
+# Execute SiteConfigGen to generate install CRs based on SiteConfigs
+function run_siteconfigGen {
+    local opts=$(getopt -o h,e: -l help,extra-manifest-only: -- "$@")
+    if [[ $? -ne 0 ]]; then
+        # wrong options passed, exit with help message
+        installCRs_help
+        exit 1
+    fi
+
+    eval set -- "$opts"
+    while true;do
+        case "$1" in
+            -h|--help)      installCRs_help
+                            exit 0
+                            ;;
+            -e|--ems-only)  EMS_ONLY="$2"
+                            shift 2
+                            ;;
+            # end of all options
+            --)             shift
+                            break
+                            ;;
+            *)              installCRs_help
+                            exit 1
+                            ;;
+        esac
+    done
+
+    if [ ${EMS_ONLY} != "false" ] && [ ${EMS_ONLY} != "true" ]; then
+        echo "Invalid value ${EMS_ONLY} for \"--ems-only\" flag. Expected value is true or false." >&2
+        exit 1
+    fi
+
+    outDir="/out/generated_installCRs"
```

Comment:
Maybe add a comment explaining that verify_arguments uses outDir to set destDir. The setting of destDir as a side-effect of the function is hard to track when reading through.